webapp/src/modules/parcels/selectors.js

```diff
@@ -1,7 +1,11 @@
 import { createSelector } from 'reselect'
-import { EDIT_PARCEL_REQUEST } from './actions'
+import { EDIT_PARCEL_REQUEST, FETCH_PARCEL_REQUEST } from './actions'
 import { getPublications as getAllPublications } from 'modules/publication/selectors'
 import { buildCoordinate } from 'lib/utils'
+import { isLoadingType } from 'modules/loading/selectors'
+import { getMappedMortgages } from 'modules/mortgage/selectors'
+// import { getAddress } from 'modules/wallet/selectors'
```

Comment:
Whops commented out code

Reply:
Oooohhh!!! Thanks you noticed. It was a selector imported as undefined. I forgot to revisited

Reply:
☝️